fighters/purin/src/acmd/aerials.rs

```diff
@@ -24,13 +24,8 @@ unsafe fn purin_attack_air_n_game(fighter: &mut L2CAgentBase) {
     if is_excute(fighter) {
         FT_MOTION_RATE(fighter, 1.125);
         AttackModule::clear_all(boma);
-    }
```

Comment:
Not sure why this change was made to nair, it's not listed in the changelog. This makes nair autocancel as soon as its hitboxes clear and extends its endlag